rust-crates/symblib-capi/c/symblib.h

```diff
@@ -138,6 +138,44 @@ extern SymblibStatus symblib_retpadextr_submit(
 // Frees a return pad extractor.
 extern void symblib_retpadextr_free(SymblibRetPadExtractor* extr);
 
+// Opaque handle to SymblibPointResolver.
+typedef struct SymblibPointResolver SymblibPointResolver;
+
+// Create a new SymblibPointResolver handler from a Go executable.
```

Comment:
No need to mention Go here as there's nothing specific to Go, it's a C API that can be called from any language that can interface with C libraries.

```suggestion
// Creates a new SymblibPointResolver handler.
```

Reply:
Resolved with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/commit/90250132bbe392ae7ed3b4e64023b581a11d3399

Reply:
I think this was a misunderstanding? The function's doc comment referred to the sort of executable that the point resolver can symbolize, not the language that this function can be called from?